docs/features/UNIFIED_CANVAS.md

```diff
@@ -0,0 +1,96 @@
+* **And more**: There are a number of creative ways to use the Canvas, and the above are just starting points. We're excited to see what you come up with!
+
+
+# Generation Methods
+The Canvas can use all generation methods available (Txt2Img, Img2Img, Inpainting, and Outpainting), and these will be automatically selected and used based on the current selection area within the Bounding Box.
+
+### Text to Image -
+If the Bounding Box is placed over an area of canvas with an **empty Base Layer**, invoking a new image will use **Text to Image**. This generates an entirely new image based on your prompt.
+
+### Image to Image - 
+If the Bounding Box is placed over an area of canvas with an **existing Base Layer area with no transparent pixels or masks**, invoking a new image will use **Image to Image**. This uses the image within the bounding box and your prompt to interpret a new image. The image will be closer to your original image at lower Image to Image strengths.
+
+### Inpainting - 
+If the Bounding Box is placed over an area of canvas with an **existing Base Layer and any pixels selected using the Mask layer**, invoking a new image will use **Inpainting**. Inpainting uses the existing colors/forms in the masked area in order to generate a new image for the masked area only. The unmasked portion of the image will remain the same. Image to Image strength applies to the inpainted area.
+
+If you desire something completely different than the original image in your new generation (i.e., ignoring existing colors/forms), consider toggling the Inpaint Replace setting on, and use high values for both that and Image to Image strength.
+
+Note: By default, the **Scale Before Processing** option will be set to automatically generate at a larger resolution and scale down to inpaint more coherent details. In order to leverage this feature, the best inpainting results will be found by resizing the bounding box to a smaller area that contains your mask, and updating your prompt to describe *just* the area within the bounding box.
+
+### Outpainting - 
+If the Bounding Box is placed over an area of canvas partially filled by an existing Base Layer area and partially by transparent pixels or masks, invoking a new image will use **Outpainting**, as well as **Inpainting** any masked areas.
+ 
+
+# Advanced Features
+
+Features with non-obvious behavior are detailed below, in order to provide clarity on the intent and common use cases we expect for utilizing them.
+
+## Toolbar
+
+### Mask Options
+* **Enable Mask** - This flag can be used to Enable or Disable the currently painted mask. This allows you the option to maintain your mask layer without it affecting current invocations.
+* **Preserve Masked Area** - When enabled, Preserve Masked Area inverts the inpainting process, maintaining the masked areas and regenerating unmasked areas.
```

Comment:
Note for future UIs: This should be a radio toggle or a popup with two choices: "Modify Masked Area" and "Preserve Masked Area". By making it explicitly binary, people will more likely grok it.